### ISSUE TITLE
Checkbox can't set value width.

### ISSUE THREAD

**Issue by khanghyxda** [flutter/flutter#19705]
Why checkbox can't set value width. If I want to change checkbox width then only have a way to change the checkbox file in the flutter framework. That not true way. Can put width of checkbox is properties not constant.

**Comment by zoechi** [comment]
The widget is designed to match the Material Design spec.
Why would you want to change the checkbox width?

**Comment by zoechi** [comment]
Makes sense.
As a workaround you can still copy the source and adjust it do your needs.

**Comment by jonahwilliams** [comment]
Checkbox has a tap area of 48 dp by 48 dp.  To make this more obvious you could put a border around it.

**Comment by khanghyxda** [comment]
Thanks you. :).

**Comment by lukepighetti** [comment]
I don't see anywhere in the spec that specifies a checkbox size. https://material.io/develop/web/components/input-controls/checkboxes#checkboxes

I only see that there should be a minimum tap target of `48px` square.

Flutter `Checkbox` does not meet Material spec.

**Comment by lukepighetti** [comment]
Looks like Material spec is 24px based on https://t.co/GIkpbveao2?amp=1
